tools/webpack.config.ts

```diff
@@ -178,6 +179,11 @@ export default (env?: string): webpack.Configuration[] => {
                     sourceMap: true,
                 }),
             ],
+            splitChunks: {
+                chunks: "all",
+                maxAsyncRequests: 20,
+                maxInitialRequests: 20,
```

Comment:
Can you add a comment on why these parameters should be 20?   I assume it should be 5 if you're not using HTTP 2, but we are?

Reply:
Right, these numbers came from [webpack/examples/many-pages/webpack.config.js](https://github.com/webpack/webpack/blob/v4.41.2/examples/many-pages/webpack.config.js) with `// for HTTP2` and seemed pretty reasonable. In practice it looks like we aren’t actually hitting the max so the exact numbers won’t matter as long as they’re large enough.